app/src/main/java/org/kiwix/kiwixmobile/utils/KiwixDialog.kt

```diff
@@ -1,17 +1,19 @@
 package org.kiwix.kiwixmobile.utils
 
+import android.net.wifi.WifiConfiguration
 import org.kiwix.kiwixmobile.R
 import org.kiwix.kiwixmobile.zim_manager.fileselect_view.adapter.BooksOnDiskListItem.BookOnDisk
 
 sealed class KiwixDialog(
   val title: Int?,
   val message: Int,
   val positiveMessage: Int,
-  val negativeMessage: Int
+  val negativeMessage: Int?,
+  val neutralMessage: Int?
```

Comment:
KiwixDialog should not have the neutralMessage field

Reply:
Fixed